CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 2016.10.11
+* `Releaf.application.config.i18n_database.create_missing_translations`
+  config renamed to `Releaf.application.config.auto_creation`
```

Comment:
That's a weird config name. The name alone doesn't say what it's going to create automatically.